components/popover/index.css

```diff
@@ -10,136 +10,354 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-@import '../overlay/index.css';
+@import "../commons/overlay-coretokens.css";
 
 .spectrum-Popover {
-  --spectrum-popover-target-offset: 13px;
-  --spectrum-popover-dialog-padding: 30px 29px;
-  --spectrum-popover-dialog-min-width: 270px;
+  --spectrum-popover-background-color: var(--spectrum-background-layer-2-color);
+  --spectrum-popover-border-width: var(--spectrum-border-width-100);
+  --spectrum-popover-border-color: var(--spectrum-gray-400);
 
-  --spectrum-popover-min-width: var(--spectrum-global-dimension-size-400);
-  --spectrum-popover-min-height: var(--spectrum-global-dimension-size-400);
+  /* popover inner padding */
+  --spectrum-popover-content-area-spacing-vertical: var(--spectrum-popover-top-to-content-area);
 
-  /* Popover tip positioning for left/right oriented popovers (-11px/-25px) */
-  --spectrum-popover-tip-horizontal-offset: calc(var(--spectrum-global-dimension-size-250) * -1);
-  /* Popover tip positioning for top/bottom oriented popovers (6px/8px) */
-  --spectrum-popover-tip-vertical-offset: var(--spectrum-global-dimension-size-75);
+  /* popover drop shadow */
+  --spectrum-popover-shadow-horizontal: var(--spectrum-drop-shadow-x);
+  --spectrum-popover-shadow-vertical: var(--spectrum-drop-shadow-y);
+  --spectrum-popover-shadow-blur: var(--spectrum-drop-shadow-blur);
+  --spectrum-popover-shadow-color: var(--spectrum-drop-shadow-color);
 
-  --spectrum-popover-tip-triangle-top-offset: calc(var(--spectrum-global-dimension-size-225) * -1);
+  /* popover corner radius */
+  --spectrum-popover-corner-radius: var(--spectrum-corner-radius-100);
+
+  /* pointer tip dimensions */
+  --spectrum-popover-pointer-width: var(--spectrum-popover-tip-width);
+  --spectrum-popover-pointer-height: var(--spectrum-popover-tip-height);
+
+  /* pointer tip - default spacing to edge - corner radius plus half of tip width to neutralize override */
+  --spectrum-popover-pointer-edge-offset: calc(var(--spectrum-corner-radius-100) + (var(--spectrum-popover-tip-width) / 2));
+
+  /* pointer tip - spacing to edge will center pointer to source - apply in markup by setting '--spectrum-popover-pointer-edge-offset' value == half of source */
+  --spectrum-popover-pointer-edge-spacing: calc(var(--spectrum-popover-pointer-edge-offset) - (var(--spectrum-popover-tip-width) / 2));
+}
+
+/* windows high contrast mode */
+@media (forced-colors: active) {
+  .spectrum-Popover {
+    --spectrum-popover-border-color: CanvasText;
+  }
 }
 
 .spectrum-Popover {
-  @inherit: %spectrum-overlay;
+   @inherit: %spectrum-overlay;
```

Comment:
nit:space

Reply:
Eagle eye! Thank you! Fixed in latest commit.